### PR TITLE
Corrected issues from #151 and fixed shellcheck problems

### PR DESCRIPTION
**Pull request by reaperzn**

### CODE REVIEW

**Comment by reaperzn** [comment]
Resolves #151

File: auter.aptModule [hunk 1]
```diff
@@ -26,23 +26,25 @@ function prepare_updates() {
...
+      AVAILABLE_PACKAGE_COUNT=$("${PACKAGE_MANAGER}" -u upgrade --assume-no "${PACKAGEMANAGEROPTIONS[@]}" | awk '/upgraded,.*newly installed,/ {sum=$1+$3} END {print sum}')
 
       if [[ ${ERROR_COUNT} -eq 0 ]]; then
         # If there are packages to be installed then download them.
         if [[ "${AVAILABLE_PACKAGE_COUNT}" -gt 0 ]]; then
           sleep $((RANDOM % MAXDELAY))
           if [[ "${ONLYINSTALLFROMPREP}" == "yes" ]]; then
             [[ -d "${DOWNLOADDIR}/${CONFIGSET}" ]] || mkdir -p "${DOWNLOADDIR}/${CONFIGSET}"
-            DOWNLOADOPTION="-o dir::cache::archives=${DOWNLOADDIR}/${CONFIGSET}"
+            DOWNLOADOPTION=("-o dir::cache::archives=${DOWNLOADDIR}/${CONFIGSET}")
```
Comment (rhodesn) [inline auter.aptModule]: This needs to be:
```sh
DOWNLOADOPTION=("-o" "dir::cache::archives=${DOWNLOADDIR}/${CONFIGSET}")
```
To separate the two into individual array indexes. If we don't do this then apt will download updates to `/var/cache/apt/archives` instead.

**Review by rhodesn — CHANGES_REQUESTED** [review]

File: auter.aptModule [hunk 2]
```diff
@@ -71,45 +73,50 @@ function prepare_updates() {
...
       sleep $((RANDOM % MAXDELAY))
       logit "INFO: Applying updates"
-      local PACKAGES_BEFORE=$(dpkg --list)
+      PACKAGES_BEFORE=$(dpkg --list)
 
       # We don't want to allow the user to interrupt a yum/dnf/apt transaction or Bad Things Happen.
       echo "Trying to update"
       trap '' SIGINT SIGTERM
-      RUN_OUTPUT=$("${PACKAGE_MANAGER}" "${UPDATEACTION}" "${PACKAGEMANAGEROPTIONS[@]}" "${DEBS}")
+      RUN_OUTPUT=$("${PACKAGE_MANAGER}" "${UPDATEACTION}" "${PACKAGEMANAGEROPTIONS[@]}" -y "${DEBS}")
```
Comment (rhodesn) [inline auter.aptModule]: Need to unquote `"${DEBS}"` as it's returning:
```sh
E: Unsupported file /var/cache/auter/default/*.deb given on commandline
```

**Review by rhodesn — APPROVED** [review]

**Comment by rhodesn** [comment]
Tested the changes and it all looks good.

**Pull request merged**